metadata-ws/src/test/resources/features/study.feature

```diff
@@ -60,26 +90,30 @@ Feature: study object
     },
     "name": "test chimpanzee study",
     "deprecated": false,
-    "releaseDate": today,
-    "taxonomy": "TAXONOMY_5"
+    "releaseDate": today
     """
     Then set the URL to STUDY3
 
+    # Create analyses to link studies to taxonomies
+    When I create an analysis with Analysis1 for accession, REFERENCE_SEQUENCE1 for reference sequence and STUDY1 for study
+    When I create an analysis with Analysis2 for accession, REFERENCE_SEQUENCE2 for reference sequence and STUDY2 for study
+    When I create an analysis with Analysis3 for accession, REFERENCE_SEQUENCE3 for reference sequence and STUDY3 for study
+
     When I request elaborate search for the studies base <base> and with the parameters: <query>
     Then the response code should be 200
     And the response should contain <N> studies
     And the href of the study of studies has items <url>
 
     Examples:
-      | base | query | N | url |
-      | taxonomy-id | id=9606 | 1 | STUDY1 |
-      | taxonomy-id | id=9596 | 2 | STUDY2,STUDY3 |
-      | taxonomy-id | id=207598 | 3 | STUDY1,STUDY2,STUDY3 |
-      | taxonomy-id | id=0 | 0 | NONE |
-      | taxonomy-name | name=Homo sapiens | 1 | STUDY1 |
-      | taxonomy-name | name=Pan | 2 | STUDY2,STUDY3 |
-      | taxonomy-name | name=Homininae | 3 | STUDY1,STUDY2,STUDY3 |
-      | taxonomy-name | name=None | 0 | NONE |
+      | base          | query             | N | url                  |
```

Comment:
Can we have a test for direct search as well, like fetching the taxonomy of a given study ?

Reply:
Expanded the first scenario in `study.feature` file to fetch & check taxonomy of a study directly. This scenario is now called “register a study successfully and check its fields”.